rbac/providers/azure/aad_auth.py

```diff
@@ -35,10 +38,11 @@ def __init__(self):
 
     def get_token(self):
         """Get an access Token for Azure Active Directory Graph API."""
-        data = {'client_id': CLIENT_ID,
-                'scope': 'https://graph.microsoft.com/.default',
-                'client_secret': CLIENT_SECRET,
-                'grant_type': 'client_credentials'}
+        data = {'grant_type': GRANT_TYPE,
```

Comment:
So I thought we were going to do two different auth methods.  One for certs and one for Client Secrets.  I don't know if this works for both?  Have you checked?